x/pubkey/client/cli/tx.go

```diff
@@ -50,11 +51,11 @@ func AddKey(ac address.Codec) *cobra.Command {
 			}
 			serverCfg := server.GetServerContextFromCmd(cmd).Config
 
-			fromAddr := clientCtx.GetFromAddress()
-			if fromAddr.Empty() {
+			valAddr := sdk.ValAddress(clientCtx.GetFromAddress())
```

Comment:
I'm confused how we can change the address format here in the CLI, but we don't need changes in the msg_server of the Pubkey module?

Reply:
So this is just wrapping the underlying bytes. I suppose we could use the bytes directly for the message, but that would reduce readability.